contrib/native/client/CMakeLists.txt

```diff
@@ -88,19 +88,28 @@ if(MSVC)
     set(Boost_USE_STATIC_LIBS ON)
     set(Boost_USE_MULTITHREADED ON)
     set(Boost_USE_STATIC_RUNTIME OFF)
+elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+    set(Boost_USE_STATIC_LIBS ON)
+    set(Boost_USE_MULTITHREADED ON)
+    #    To build a production version, the macos build must use a shaded version
+    #    of boost. Arbirtarily, we choose the new namspace to be drill_boost.
+    #    See the instructions in the readme for the mac and rebuild boost. Then
+    #    uncomment the line below to build
+    #    set(Boost_NAMESPACE drill_boost)
```

Comment:
There is no change needed to the source. The bcp option namespace-alias aliases the boost namespace to the namespace you provide, so no code change is needed. (Someone at boost thought this thru!) 
Added a linux portion. Can you please doubecheck?